src/commands/moderation/timeout.ts

```diff
@@ -105,11 +115,20 @@ const modLogEmbed = (moderator: User, user: User, reason: string, timeoutDuratio
             name: 'Duration',
             value: timeoutDurationInEnglish(timeoutDuration),
         },
+        {
+            name: 'Date',
+            value: `${formattedDate} Z`,
+        },
     ],
-    timestamp: Date.now(),
     footer: { text: `User ID: ${user.id}` },
 });
 
+const warnFailed = makeEmbed({
+    title: 'Warn - Failed',
+    description: 'Failed to warn user, doc not saved to mongoDB',
```

Comment:
```suggestion
    description: 'Failed to warn user as part of the timeout, warning is not saved to mongoDB',
```
Just a small cleanup in language to make it clear it is part of the timeout procedure